### ISSUE TITLE
Option to disable blur on lockscreen does nothing

### ISSUE THREAD

**Issue by pacho2** [aunetx/blur-my-shell#91]
I was using https://github.com/PRATAP-KUMAR/Control_Blur_Effect_On_Lock_Screen and it effectively drops the lock screen blur properly.

I later noticed that, after enabling blur my shell extension, it was blurring my screen again after resuming from suspend. I think it's due to it still trying to mess with blur when extension is disabled at suspend time as it was happening with Bing extension blur functionality:
https://github.com/neffo/bing-wallpaper-gnome-extension/issues/113

I then discovered that there was another issue: blur-my-shell looks to have an option to enable/disable blur in lockscreen, but it doesn't work at all :/ It changes nothing even without suspending/resuming. 

In summary:
- lockscreen option looks to have no effect and always leave the default gnome blurred lockscreen.
- Blur-my-shell looks to collide with other extensions handling blur as explained in https://github.com/neffo/bing-wallpaper-gnome-extension/issues/113 , that was fixed simply not touching blur at all if the option to touch blur is not activated at all (leaving it to other extensions like Control_Blur_effect or Bing one to do that). This is specially noticeable closing laptop lid and going to suspend -> resume cycle, as Blur-my-shell extension will change blur setting when it is getting disabled always, even if user doesn't want it to touch the lock screen blur.

Thanks a lot

**Comment by aunetx** [comment]
Thanks for the issue! I noticed that the blur option does nothing too, but it should...

The 'blur lockscreen' option was actually not whether or not to blur the lockscreen, but to change brightness/sigma intensity so the lockscreen blur matches the overview/panel blur. Disabling this option should not remove the blur, but reset it so Blur my Shell does not touch to the lockscreen blur, allowing other extensions (like the one you mentioned) to change the blur intensity without interfering with Blur my Shell.

The problem is that not only the blur is not changed properly on lockscreen, but it interferes with other extensions even when deactivated... So I will try to change this, and make this part of the extension work too!

Thanks again for the issue :)

**Comment by aunetx** [comment]
Should now be fixed! Please tell me if that works for you :)

However, I can't find a way to blur the lockscreen after sleep, as the extension is disabled by gnome-shell during suspend... I will search a way to do this, it would improve a lot the lock behaviour of Blur my Shell

**Comment by pacho2** [comment]
I will try it as soon as possible

Regarding how to fix the blur after resume... I think it's not possible. The workaround other extensions apply to not mess the blur settings is to drop the offending disable statements... but that can only be solved in the github versions, as extensions.gnome.org moderators will enforce the disable to work for the versions there.

I ignore if there is any upstream report to suggest to not call disable for shell extensions when going to suspend (I still don't understand why that is done as, supposedly, when we suspend/hibernate we want to get the system "frozen" in its previous state :/

**Comment by pacho2** [comment]
The patch fixes the setting and now lockscreen follows the blur/brightness from blur extension... but it applies the changes always, even if the lockscreen tick is disabled :/

But thanks for the really fast response :D

**Comment by pacho2** [comment]
The way to "solve" it in Bing extension to not collide with other extensions is here:
https://github.com/neffo/bing-wallpaper-gnome-extension/commit/d6350119bccb896744e574d0f10e34c1309bc1b1

The idea is to not touch at all lockscreen settings when the tick is not enabled
